From: Steve French <stfrench@microsoft.com>
Date: Wed, 13 Mar 2019 16:48:33 -0500
Subject: [PATCH] cifs: remove unused value pointed out by Coverity
Git-commit: d44d13723b0994bc61e0f2d3efeb17856128673b
Patch-mainline: v5.1-rc1
References: bsc#1144333

Detected by CoverityScan CID#1438719 ("Unused Value")

buf is reset again before being used so these two lines of code
are useless.

Signed-off-by: Steve French <stfrench@microsoft.com>
Reviewed-by: Ronnie Sahlberg <lsahlber@redhat.com>
Acked-by: Aurelien Aptel <aaptel@suse.com>
---
 fs/cifs/connect.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/fs/cifs/connect.c b/fs/cifs/connect.c
index b95db2b593cb..a8e9738db691 100644
--- a/fs/cifs/connect.c
+++ b/fs/cifs/connect.c
@@ -1191,10 +1191,6 @@ cifs_demultiplex_thread(void *p)
 			continue;
 		}
 
-		if (server->large_buf)
-			buf = server->bigbuf;
-
-
 		server->lstrp = jiffies;
 
 		for (i = 0; i < num_mids; i++) {
-- 
2.16.4

